app/Http/Controllers/UsersController.php

```diff
@@ -20,14 +21,14 @@ public function index($id = null) {
         }
     }
     public function uploadImage(Request $request) {
-        if($request->file('file')!==null) :
+        define('width', '426');
```

Comment:
You should read this. Almost meaningless if you define here.
http://php.net/manual/en/language.constants.php